sdk/containerservice/Azure.ResourceManager.ContainerService/src/autorest.md

```diff
@@ -190,21 +193,6 @@ prepend-rp-prefix:
   - TrustedAccessRoleRule
   - TrustedAccessRoleBindingProvisioningState
 
-operations-to-lro-api-version-override:
```

Comment:
Removing `2017-08-31` means this works for all versions?

Reply:
Right~